source/uwp/Renderer/lib/AdaptiveCard.cpp

```diff
@@ -40,7 +40,7 @@ namespace AdaptiveNamespace
         std::string adaptiveJsonString;
         RETURN_IF_FAILED(HStringToUTF8(adaptiveJson, adaptiveJsonString));
 
-        return FromJsonString(adaptiveJsonString, elementParserRegistration, actionParserRegistration, parseResult);
+        return _FromJsonString(adaptiveJsonString, elementParserRegistration, actionParserRegistration, parseResult);
```

Comment:
>_ [](start = 15, length = 1)

This is not a convention we've been using in general in our code, and it looks like our guidelines are against it ("Do not prefix methods with a scoping prefix.")